install-omb.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+cd $HOME
+source ~/wget_check.sh
```

Comment:
Why is wget_check.sh inside the homedir? Shouldn't this code snippet look more like:
```
source wget_check.sh
cd $HOME
```